app/src/main/res/layout/feedinfo.xml

```diff
@@ -123,24 +87,103 @@
                     android:maxLines="4"
                     android:paddingTop="4dp"
                     android:paddingBottom="4dp"
-                    tools:text="http://www.example.com/feed"
-                    tools:background="@android:color/holo_green_dark"/>
+                    tools:background="@android:color/holo_green_dark"
+                    tools:text="http://www.example.com/feed" />
 
             <TextView
-                    style="@style/AntennaPod.TextView.Heading"
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:layout_marginTop="4dp"
-                    android:layout_marginBottom="8dp"
-                    android:text="@string/description_label"/>
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:textSize="18sp"
+                android:layout_marginTop="16dp"
+                android:layout_marginBottom="4dp"
+                android:text="@string/description_label"
+                android:textColor="?android:attr/textColorPrimary"
+                tools:background="@android:color/holo_red_light" />
 
             <TextView
                     android:id="@+id/txtvDescription"
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:textIsSelectable="true"
                     android:text="@string/design_time_lorem_ipsum"
-                    tools:background="@android:color/holo_green_dark"/>
+                    android:textIsSelectable="true"
+                    tools:background="@android:color/holo_green_dark" />
+
+            <TextView
+                android:id="@+id/lblStatistics"
```

Comment:
Please use the same indention as the lines above (all new lines that you added to the file). 

Current:

```
<TextView
    android:id="@+id/lblStatistics"
```

More consistent:

```
<TextView
        android:id="@+id/lblStatistics"
```